playbooks/roles/homebrew/tasks/main.yml

```diff
@@ -4,6 +4,11 @@
 
 - name: Install Homebrew
   shell: ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"
+  homebrew_tap:
+    name:
+    - homebrew/bundle
+    - homebrew/cask
+    - homebrew/service
```

Comment:
```suggestion
    - homebrew/services
```

Reply:
:heavy_check_mark: Whoops! Thanks for catching that!